persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/CassandraPersistence.java

```diff
@@ -115,7 +115,7 @@ public void initialize(Config config) {
 
     daemon.start();
 
-    waitForSchema(StorageService.RING_DELAY);
+    waitForSchema(5 * StorageService.RING_DELAY);
```

Comment:
This fixes the schema agreement issue because it waits (a maximum, probably less) 2.5 minutes which is guaranteed to be longer than the migration of schema which can be 2 minutes in some cases. Anyway, I have a follow up ticket for this called: "Schema agreement times out for DDL queries (usually after startup)".